$OpenBSD$

Don't reconstruct the ps file from the PDF, this makes ghostscript shit itself:
GPL Ghostscript 9.04: Error: Font Renderer Plugin ( FreeType ) return code = -1

--- doc/makefile.in.orig	Wed Apr 18 22:09:04 2012
+++ doc/makefile.in	Wed Apr 18 22:09:11 2012
@@ -91,7 +91,7 @@ all-pre:
 
 all-post:
 
-doc: info pdf html txt ps
+doc: info pdf html txt
 
 info: gambit-c.info
 
